pkg/scheduler/framework/interface.go

```diff
@@ -718,6 +718,9 @@ type PreFilterResult struct {
 	// The set of nodes that should be considered downstream; if nil then
```

Comment:
perhaps outside of the scope of this PR (something we can fix as followup), but a question for @AxeZhan, @alculquicondor: Does it make sense to share this result type between `PreFilter` and `RunPreFilterPlugins`? As now, there is a field which users of the framework should never need to set

Reply:
Could you please elaborate? I don't quite get it.

Reply:
@sanposhiho

Reply:
Yes I agree: we suggested two similar things at two different places https://github.com/kubernetes/kubernetes/pull/122251#discussion_r1638811772, right?